src/main/java/com/powsybl/openloadflow/network/impl/LfAreaImpl.java

```diff
@@ -17,9 +17,8 @@
 /**
  * @author Valentin Mouradian {@literal <valentin.mouradian at artelys.com>}
  */
-public class LfAreaImpl extends AbstractPropertyBag implements LfArea {
+public class LfAreaImpl extends AbstractElement implements LfArea, PropertyBag {
```

Comment:
AbstractElement already implements PropertyBag

```suggestion
public class LfAreaImpl extends AbstractElement implements LfArea {
```

Reply:
Actually LfElement also extends directly PropertyBag somehow